onnx/utils.py

```diff
@@ -23,26 +23,23 @@ def __init__(self, model: ModelProto) -> None:
     def _build_name2obj_dict(objs):  # type: ignore
         return {obj.name: obj for obj in objs}
 
-    def _collect_new_io_core(self, original_io, io_names_to_extract):  # type: ignore
+    def _collect_new_io_core(
+        self, original_io, io_names_to_extract
```

Comment:
Could you help add types to the inputs as well?

Reply:
Thank you @justinchuby , I add the input types
```
original_io: RepeatedCompositeFieldContainer[ValueInfoProto],
io_names_to_extract: list[str],
```